web/pandas/about/index.html

```diff
@@ -83,4 +83,4 @@
 Is in the core of _pandas_ to be respectful and welcoming with everybody,
 users, contributors and the broader community. Regardless of level of experience,
 gender, gender identity and expression, sexual orientation, disability,
-personal appearance, body size, race, ethnicity, age, religion, or nationality.
+personal appearance, body size, race, ethnicity, age, religion, or nationality.
```

Comment:
@datapythonista do we care about getting an extra newline here?

Reply:
I don't think it's important, but let's better avoid unrelated changes. Can you restore the original version of this file please?

Reply:
ok